compose.js

```diff
@@ -21,20 +60,29 @@ export const merge = (dst, src) => mergeWith(dst, src, (dstValue, srcValue) => {
  */
 function createFactory(descriptor) {
   return function Stamp(options = {}, ...args) {
-    let obj = Object.create(descriptor.methods || {});
+    // The 'methods' metadata object becomes the prototype for new object instances.
+    const obj = Object.create(descriptor.methods || {});
```

Comment:
Could be also named `instance` instead of just generic `obj` to clearly represent what is it about.